### ISSUE TITLE
Lag and issues typing the "(" character in the iPython console

### ISSUE THREAD

**Issue by bcaddy** [spyder-ide/spyder#8502]
### Issue Report Checklist

* [x] Searched the [issues page](https://github.com/spyder-ide/spyder/issues?q=is%3Aissue) for similar reports
* [x] Read the relevant sections of the [Spyder Troubleshooting Guide](https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ) and followed its advice
* [x] Reproduced the issue after updating with ``conda update spyder`` (or ``pip``, if not using Anaconda)
* [x] Could not reproduce inside ``jupyter qtconsole`` (if console-related)
* [x] Tried basic troubleshooting (if a bug/error)
    * [x] Restarted Spyder
    * [x] Reset preferences with ``spyder --reset``
    * [x] Reinstalled the latest version of [Anaconda](https://www.anaconda.com/download/)
    * [x] Tried the other applicable steps from the Troubleshooting Guide
* [ ] Completed the **Problem Description**, **Steps to Reproduce** and **Version** sections below

## Problem Description
When using the ipython console there are enormous amounts of lag at seemingly random times.  It can happen with importing libraries, ls and cd commands, or just using the help() function.  The biggest offender seems to be when typing the left parenthetical character "(".  When this is typed the lag can often run into multiple minutes.  When I turn off Automatic Connection from the console to the help pane the problem with left parentheticals seems to go away.  The issue persists across all versions of Spyder from 3.2.8 on (I didn't check earlier ones) even though it didn't used to happen with 3.2.8.  

### What steps reproduce the problem?

1. Try typing any functions left parenthetical with Automatic connection turned on

### What is the expected output? What do you see instead?
The expected output is that the docstrings show up in the help pane.  Instead they don't and long delays are encountered before the parenthetical even shows up.

```

## Versions
<!--- You can get this information from Help > About Spyder...
or (if Spyder won't launch) the "conda list" command
from the Anaconda Prompt/Terminal/command line. --->

* Spyder version: 3.2.8 through 3.3.2
* Python version: 3.6.7
* Qt version: 5.9.7
* PyQt version: 5.9.2
* Operating System name/version: MacOS Mojave 10.14.2

**Comment by bcaddy** [comment]
It happens when I have a completely empty console.  It will even happen on the very first line of the console.

**Comment by ccordoba12** [comment]
Does it happen when the options I mentioned are turned off?

**Comment by bcaddy** [comment]
Yes.  There might be two issues here that are pretending to be one.  The issue with the "(" not showing up appear to be fixed by this problem but I'm still getting lots of lag with simple commands, especially ipython specific ones (pwd, mv, cd, etc).  They keep trying to execute as part of a python statement instead of as terminal commands.  Aka I will type "pwd" then hit enter and it will act like it's waiting for the next line of code with the "...:" prompt

**Comment by ccordoba12** [comment]
For the second one, please update qtconsole and try again.

**Comment by bcaddy** [comment]
That seems to fix it.  The help pane still has issues though.

**Comment by ccordoba12** [comment]
As I said, that's a known issue and we don't have a solution for it yet (other than deactivating the connection between the console and Help).
